### ISSUE TITLE
build a shape model from test TPFs fail with singular matrix error

### ISSUE THREAD

**Issue by jorgemarpa** [SSDataLab/psfmachine#15]
When runing `.build_shape_model()' using the testing TPF it fails.

```python
import lightkurve as lk
import psfmachine as psf
from astropy.utils.data import get_pkg_data_filename

c = psf.TPFMachine.from_TPFs(tpfs)
c.build_shape_model(plot=True)
```

Error Traceback:
```python
/Users/jorgemarpa/Work/BAERI/ADAP/psfmachine/src/psfmachine/machine.py:288: RuntimeWarning: Mean of empty slice
  mean_flux = np.nanmean(self.flux[self.time_mask], axis=0)
---------------------------------------------------------------------------
LinAlgError                               Traceback (most recent call last)
<ipython-input-5-b5ee10b7ab0e> in <module>
----> 1 c.build_shape_model(plot=True)

~/Work/BAERI/ADAP/psfmachine/src/psfmachine/machine.py in build_shape_model(self, plot, flux_cut_off)
    766         # Mask of shape nsources x number of pixels, one where flux from a
    767         # source exists
--> 768         self._get_source_mask()
    769         # Mask of shape npixels (maybe by nt) where not saturated, not faint,
    770         # not contaminated etc

~/Work/BAERI/ADAP/psfmachine/src/psfmachine/machine.py in _get_source_mask(self, upper_radius_limit, lower_radius_limit, upper_flux_limit, lower_flux_limit, plot)
    334             sigma_w_inv = A[k].T.dot(A[k])
    335             B = A[k].T.dot(f[temp_mask][k])
--> 336             w = np.linalg.solve(sigma_w_inv, B)
    337             res = np.ma.masked_array(f[temp_mask], ~k) - A.dot(w)
    338             k &= ~sigma_clip(res, sigma=3).mask

<__array_function__ internals> in solve(*args, **kwargs)

~/.pyenv/versions/adap/lib/python3.8/site-packages/numpy/linalg/linalg.py in solve(a, b)
    391     signature = 'DD->D' if isComplexType(t) else 'dd->d'
    392     extobj = get_linalg_error_extobj(_raise_linalgerror_singular)
--> 393     r = gufunc(a, b, signature=signature, extobj=extobj)
    394 
    395     return wrap(r.astype(result_t, copy=False))

~/.pyenv/versions/adap/lib/python3.8/site-packages/numpy/linalg/linalg.py in _raise_linalgerror_singular(err, flag)
     86 
     87 def _raise_linalgerror_singular(err, flag):
---> 88     raise LinAlgError("Singular matrix")
     89 
     90 def _raise_linalgerror_nonposdef(err, flag):

LinAlgError: Singular matrix
```

**Comment by jorgemarpa** [comment]
The testing TPFs only have 10 cadences (first 10). The problem is `c.time_mask` is false for all those cadences, therefore `mean_flux` is nan everywhere. 
We have to update the testing TPFs with valid cadences.

**Comment by jorgemarpa** [comment]
After a deeper investigation of this, I found the problem is not with the cadences saved in the test TPFs. The times for these TPFs get masked by `TPFmachine` here:
https://github.com/SSDataLab/psfmachine/blob/ebd4639bc60dffaf48eb63116c21dcfd3c864b24/src/psfmachine/tpf.py#L66

This can be avoided by saving more cadences in the TPFs.
I created a new set of test TPFs with 500 cadences. These allow building the shape and time models with default parameters.
But now each TPF is ~500 kb, a total of 5 MB for the testing set. 
I think is worth keeping this new set. It will allow adding more unit tests for other functionalities of `psfmachine` in the future.
